packages/core/src/services/prompts/run.ts

```diff
@@ -29,6 +29,12 @@ export async function runPrompt({
   }
 
   const chain = createChain({ prompt: metadata.resolvedPrompt, parameters })
-  const run = await runChain({ workspace, chain, providersMap, source })
+  const run = await runChain({
+    workspace,
+    chain,
+    providersMap,
+    source,
+    persistErrors: false,
```

Comment:
This is an interesting case. We just run the prompt, we show the error in the evaluation playground but is not associated with a document or evaluation so no error to persist. 

Please confirm this is what we want

Reply:
Is it not associated with the document though?

Reply:
Talked offline. We don't have logs for evaluations in evaluations playground. So we can not associate errors with any entity in DB